crates/synthesis/src/two_qubit_decompose.rs

```diff
@@ -1268,13 +1264,11 @@ impl TwoQubitWeylDecomposition {
     }
 }
 
-type TwoQubitSequenceVec = Vec<(Option<StandardGate>, SmallVec<[f64; 3]>, SmallVec<[u8; 2]>)>;
+type TwoQubitSequenceVec = Vec<(PackedOperation, SmallVec<[f64; 3]>, SmallVec<[u8; 2]>)>;
```

Comment:
Beyond the scope of this PR, but generally I think we should get out of the habit of having these alias types. It seems like it leads to an extra layer of indirection when trying to understand the code. I'd rather see a struct type with named fields for the element `(PackedOperation, SmallVec<[f64; 3]>, SmallVec<[u8; 2]>)` and then the `Vec` be inlined where applicable.